integration/flink/src/test/java/io/openlineage/flink/FlinkContainerUtils.java

```diff
@@ -109,7 +109,8 @@ static GenericContainer<?> makeFlinkJobManagerContainer(
                     + String.format("--job-classname %s ", jobName)
                     + "--input-topics io.openlineage.flink.kafka.input1,io.openlineage.flink.kafka.input2 "
                     + "--output-topic io.openlineage.flink.kafka.output ")
-            .withEnv("FLINK_PROPERTIES", "jobmanager.rpc.address: jobmanager")
+            .withEnv(
+                "FLINK_PROPERTIES", "jobmanager.rpc.address: jobmanager\nexecution.attached: true")
```

Comment:
Please provide some comment with an explanation of `execution.attached: true`.

Reply:
Added two lines to readme. Overall, any documentation for this integration still needs to be written.